pkg/koordlet/metrics/resource_summary.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	NodeResourceAllocatable = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "node_resource_allocatable",
+		Help:      "the node allocatable of resources updated by koordinator",
+	}, []string{NodeKey, ResourceKey})
+
+	ContainerResourceRequests = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "pod_resource_requests",
```

Comment:
Should we use `container_resource_requests` or `pod_resource_requests`?

Reply:
nice catch! fixed